List selector test - static elements

Joseph-Cashmore opened the pull request:
Just static at the moment, but wanted to provide some example elements styled for incoming functionality of multiple lists.

![image](https://user-images.githubusercontent.com/9193732/98876199-ab618280-2475-11eb-8700-bd420ae8237c.png)

Commit:
moved header to separate component "ListHeader"

Commit:
Styling of header component

Any new element is static, the text is fixed for now

Commit:
logo added as component

svg stored in "static" folder and sourced into the header. Unsure if this is the appropriate folder structure for something like an svg, but made sense in my head

Commit:
lint fixes etc

empty test file removed

Commit:
list selector component created and styled

Just static at the moment, but wanted to get some example elements styled for incoming functionality

Commit:
lint fixes

Commit:
wrapper margin adjusted

This is so the last item isn't hidden behind list selector

Joseph-Cashmore committed:
Merge branch 'main' into list-selector-test

Commit:
fixed some conflicts

Commit:
mobile styling adjustments

scrolling overflow for more list buttons

may need to improve desktop version

Commit:
"hide" list selector scroll bar on desktop

Could argue this isn't good for usability, but the scroll bar looks awful on desktop, and 9 times out of 10 it will be completely unnessecary as users won't have that many lists.

Scrolling is still there, just the scroll bar can't be seen

samstradling commented on public/index.html:
```diff
@@ -27,5 +27,7 @@
     </div>
 
+    <div id="listSelector">
+    </div>
```
I guess this isn't needed anymore?

samstradling commented on src/components/ListSelector/ListSelector.js:
```diff
@@ -0,0 +1,23 @@
+      <FontAwesomeIcon
+        icon={faPlusCircle}
+        className={styles.addListIcon}
+      />
```
A test to check these list items and this button are displayed would be great!

samstradling reviewed (approved):
Nice!

Commit:
removed redundant code

Commit:
tests for list selector rendering

The test for rendering example lists doesn't feel very "complete", but this section is going to change a lot soon and so will the tests.

Commit:
updated list count test
